Photon-Tinker/LogList.swift

```diff
@@ -0,0 +1,132 @@
+//
```

Comment:
Thanks for writing this.  Imma totally steal it for Android.

Reply:
You mean the whole class? Be my guest :)